### PR TITLE
Add --mapping cli option to include more than one source file

### PR DESCRIPTION
**Pull request by remore**
This is a proposal to allow users to add more than one input source file. Please discard this if this doesn't match with the current plan or design - I can customize `cli.rb` locally for the time being (i.e. I needed to do so to make my blog app work).

### CODE REVIEW

**Comment by kateinoigakukun** [comment]
How about adding `--dir HOST_DIR::GUEST_DIR` as well as wasmtime, wasi-vfs, and `rbwasm pack`? I think it would be simpler if we passthrough the `--dir` options to `rbwasm pack`?

**Comment by remore** [comment]
That makes sense, I'll give it a try.

**Comment by remore** [comment]
@kateinoigakukun I have pushed the latest change. Is this close to what you are suggesting?

File: lib/compute_runtime/cli.rb [hunk 1]
```diff
@@ -61,18 +62,20 @@ def check_executable(command)
   class CLI
     def parse_args(args)
       args = args.dup
+      @mapping = []
       opts = OptionParser.new
       opts.on("-o", "--output FILE", "Output file") { |v| @output = v }
       opts.on("--[no-]stdlib", "Include stdlib in the output or not") { |v| @stdlib = v }
       opts.on("--remake", "Remake ruby.wasm") { |v| @remake = v }
+      opts.on("--mapping <HOST_DIR::GUEST_DIR>...", "Package a host directory into Wasm module at a guest directory") { |v| @mapping.push(v) }
```
Comment (kateinoigakukun) [inline lib/compute_runtime/cli.rb]:
```suggestion
      opts.on("--dir <HOST_DIR::GUEST_DIR>...", "Package a host directory into Wasm module at a guest directory") { |v| @mapping.push(v) }
```

File: lib/compute_runtime/cli.rb [hunk 2]
```diff
@@ -12,8 +12,9 @@ def run_rbwasm(*args)
       RubyWasm::CLI.new(stdout: $stdout, stderr: $stderr).run(args)
     end
 
-    def pack_directory(infile, outfile, dirmaps)
+    def pack_directory(infile, outfile, dirmaps, dirmap_strings)
       mapdir_args = dirmaps.collect_concat { |k, v| ["--mapdir", "#{k}::#{v}"] }
+      mapdir_args += dirmap_strings.collect_concat { |str| ["--mapdir", str] }
```
Comment (kateinoigakukun) [inline lib/compute_runtime/cli.rb]:
```suggestion
      mapdir_args += dirmap_strings.collect_concat { |str| ["--dir", str] }
```

Note that `--mapdir GUEST_DIR::HOST_DIR` is old one and `--dir HOST_DIR::GUEST_DIR` have the opposite mapping direction.

File: lib/compute_runtime/cli.rb [hunk 3]
```diff
@@ -12,8 +12,9 @@ def run_rbwasm(*args)
       RubyWasm::CLI.new(stdout: $stdout, stderr: $stderr).run(args)
     end
 
-    def pack_directory(infile, outfile, dirmaps)
+    def pack_directory(infile, outfile, dirmaps, dirmap_strings)
       mapdir_args = dirmaps.collect_concat { |k, v| ["--mapdir", "#{k}::#{v}"] }
```
Comment (kateinoigakukun) [inline lib/compute_runtime/cli.rb]:
```suggestion
      mapdir_args = dirmaps.collect_concat { |k, v| ["--dir", "#{v}::#{k}"] }
```

It's not a part of your change, but could you update here also?

**Comment by remore** [comment]
That all looks good to me, thanks for bunch of suggestions!

**Review by kateinoigakukun — APPROVED** [review]
Thanks!

**Pull request merged**